src/features/home/components/modules/video/YoutubePlayer.tsx

```diff
@@ -0,0 +1,120 @@
+import React, { ReactElement, forwardRef, useRef, useState } from 'react'
+import {
+  LayoutChangeEvent,
+  MeasureOnSuccessCallback,
+  Pressable,
+  StyleProp,
+  View,
+  ViewStyle,
+} from 'react-native'
+import Animated, { FadeOut } from 'react-native-reanimated'
+import RNYoutubeIframe, { YoutubeIframeProps, YoutubeIframeRef } from 'react-native-youtube-iframe'
+import { useTheme } from 'styled-components'
+import styled from 'styled-components/native'
+
+interface YoutubePlayerProps extends Omit<YoutubeIframeProps, 'webViewProps'> {
```

Comment:
je n'utilise plus d'`interface`, depuis que j'ai lu ça

https://www.totaltypescript.com/type-vs-interface-which-should-you-use#interfaces-can-declaration-merge

il devrait etre possible de le réécrire avec `type`

```suggestion
type YoutubePlayerProps = Omit<YoutubeIframeProps, 'webViewProps'> & {
```

Reply:
Merci pour cet article. Je l'ai lu et je suis également tombé sur cette page : 
https://www.typescriptlang.org/docs/handbook/2/everyday-types.html#differences-between-type-aliases-and-interfaces

J'ai noté ces quelques phrases : 

_Type aliases and interfaces are very similar, and in many cases you can choose between them freely._

_Interfaces may only be used to declare the shapes of objects, not rename primitives_

_Using interfaces with extends can often be more performant for the compiler than type aliases with intersections_